website/content/api-docs/system/internal-counters.mdx

```diff
@@ -897,11 +908,31 @@ $ curl \
 
 ## Read the Client Count Configuration
 
-Reading the configuration shows the current settings, as well as a flag as to whether any data can be queried.
+Reading the configuration shows the current settings, as well as a flag as to
+whether any data can be queried.
+
+- `enabled` `(string)` - returns `default-enabled` or `default-disabled` if the
+  configuration is `default`.
+- `queries_available` `(bool)` - indicates whether any usage report is
+  available. This will initially be false until the end of the first calendar
+  month after the feature is enabled.
+
+**Vault 1.14.0 or later:** The automated reporting feature exposes two
+additional read-only system settings:
+
+- `reporting_enabled` `(bool)` - indicates whether automated reporting is
+  enabled.
+- `billing_start_timestamp` `(time)` - returns the configured billing start
+  timestamp to be used for automated reporting.
+- `minimum_retention_months` `(int)` - returns the configured minimum value for `retention_months`.
+
+<Note>
+
+The `reporting_enabled` and `billing_start_timestamp` parameters are read-only
```

Comment:
This might require some line-wrapping adjustments as well, but:
```suggestion
The `reporting_enabled`,  `billing_start_timestamp`, and `minimum_retention_months` parameters are read-only
```